### PR TITLE
adds dask-funk 0.9.1

### PR DESCRIPTION
**Pull request by bmabey**

### CODE REVIEW

**Comment by synapticarbors** [comment]
You need to add `pytest-runner` to the `build` requirements since it is listed in the `setup.py` in the `setup_requires` section:

https://github.com/Savvysherpa/dask-funk/blob/master/setup.py#L13

It is available via conda-forge so that should not be an issue.

File: recipes/dask-funk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - dask >=0.8.0
+    - toolz >=0.7.4
+
+test:
+  imports:
+    - daskfunk
+
+about:
+  home: https://github.com/Savvysherpa/dask-funk
+  license: MIT
```
Comment (jakirkham) [inline recipes/dask-funk/meta.yaml]: Could add `license_family: MIT`.

File: recipes/dask-funk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  license_file: LICENSE.txt
+  summary: "Composable keyword function graphs"
+  # The remaining entries in this section are optional, but recommended
```
Comment (jakirkham) [inline recipes/dask-funk/meta.yaml]: Please drop this comment.

**Comment by bmabey** [comment]
@jakirkham thanks, I've made the suggested changes so this should be ready to be merged in.

**Comment by synapticarbors** [comment]
@bmabey - - Thanks for contributing this recipe. Soon you should get an email that invites you to join conda-forge. Once accepted you will be added to a team with the same name as this recipe. This will give you permissions on the feedstock (repo) for this recipe.

Make sure when proposing any change that you go through the typical GitHub workflow of forking the feedstock and making changes in your fork that you PR back. Once merged CIs will build and deploy any changes you make.

Please let us know if you have any questions and welcome to conda-forge.

**Pull request merged**